backend/src/nodes/nodes/image_filter/high_pass.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from . import category as ImageFilterCategory
+
+
+@NodeFactory.register("chainner:image:high_pass")
+class HighPassFilterNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply High Pass filter on image"
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Radius", minimum=0, default=1, precision=2, controls_step=1),
+            NumberInput("Contrast", minimum=0, default=1, precision=2, controls_step=1),
+        ]
+        self.outputs = [ImageOutput(image_type="Input0")]
+        self.category = ImageFilterCategory
+        self.name = "High Pass"
+        self.icon = "MdOutlineAutoFixHigh"
+        self.sub = "Miscellaneous"
+
+    def run(
+        self,
+        img: np.ndarray,
+        radius: float,
+        contrast: float,
+    ) -> np.ndarray:
+        """High Pass filter"""
+
+        if radius == 0 or contrast == 0:
```

Comment:
We already know that it's a high pass filter, so there's no need to repeat it here again :)

```suggestion
        if radius == 0 or contrast == 0:
```